cegs_portal/search/templates/search/v1/partials/_non_targeting_reo.html

```diff
@@ -1,16 +1,41 @@
 {% load humanize %}
 {% load static %}
+<style>
+    .tsv-button {
+        border: 2px solid #807e7e;
+        background-color: #fff;
+        color: #525050;
+        padding: 3px 6px;
+        text-align: center;
+        text-decoration: none;
+        display: inline-block;
+        font-size: 10px;
```

Comment:
This text is teeny tiny. It should be at least 12-14 pt (not px), I think. It's probably also helpful to have these styles in project.css.tw since they will be used on multiple pages and partials.